tox-update.ini

```diff
@@ -43,20 +43,16 @@ commands=
   # configure lvm, we exclude osd2 given this node uses lvm batch scenario (see corresponding inventory host file)
   ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/tests/functional/lvm_setup.yml --limit 'osds:!osd2'
 
-  non_container: ansible-playbook -vv -i "localhost," -c local {toxinidir}/tests/functional/dev_setup.yml --extra-vars "dev_setup=True change_dir={changedir} ceph_dev_branch={env:UPDATE_CEPH_DEV_BRANCH:master} ceph_dev_sha1={env:UPDATE_CEPH_DEV_SHA1:latest}" --tags "vagrant_setup"
+  non_container: ansible-playbook -vv -i "localhost," -c local {toxinidir}/tests/functional/dev_setup.yml --extra-vars "dev_setup=True change_dir={changedir}" --tags "vagrant_setup"
```

Comment:
take a look at how it's done in 6.0, we deploy octopus with stable-5.0, then we use stable-6.0 to upgrade.

here we have to do the same thing, first deploy pacific with ceph-ansible 6.0 and upgrade to quincy with stable-7.0